### DISCUSSION TITLE
Tessellation failed to reproduce the example5 of fepx

### DISCUSSION THREAD

**Discussion by Ming-is** [neperfepx/neper#330] [Q&A] [answered]
My neper is already version 4.2.0. When I tried to generate the Tess file of the fepx example 5 , I found that neper has been running for nearly 8 hours, but it has not been completed，At present, the computer has been running for 12 hours, and the following two instructions have not been completed successfully.The instructions are as follows：

neper -T
-n 50
-reg 1
-group "id<=25?1:2"
-ori "file(orifile,des=rodrigues)"
-oridistrib "normal(5)"
-oridescriptor "rodrigues:active"
-morpho gg
-o simulation

neper -T -n 100 -dim 3 -reg 1 -ori "file(ori,des=euler-bunge)" -oridescriptor "euler-bunge:active" -oridistrib "normal(5)" -morpho gg -domain "cube(0.005,0.005,0.005)" -statcell diameq -for tess,tesr -o simulation
The following zip file contains the orientation file I use
[ori.zip](https://github.com/rquey/neper/files/7547378/ori.zip)

**Comment by Ming-is** [comment, marked as answer]
I think I know the reason. It should be that Ubuntu has been started for too long. After restarting the virtual machine, the program runs fast.

**Reply by smallrain1996** [reply]
You can update to the latest version of neper with the following statement, which may run faster.
```
git clone https://github.com/rquey/neper.git
cd neper/src
mkdir build
cd build
cmake ..
make
sudo make install
make test
```

**Reply by Ming-is** [reply]
ok，thanks

**Reply by rquey** [reply]
Yes, likely a system problem if you are on a virtual machine (rather than a Neper problem). These commands run fine on my computer (both 4.2.0 and latest version).
